Automattic/jetpack
# Remove legacy "My Jetpack" code

None of this is used anymore with no plans on reinstating it.  The endpoint to change primary users still lives in wpcom, and will be trivial to add back if we ever want to re-add the functionality later.  


To test: 
- Build, click and do everything.  Make sure nothing looks broken and you get no errors. 
- I'm mostly concerned about the css removed, but I didn't find any usage of it in my searches.
